storage/cql_iface/cql_comm.py

```diff
@@ -1,9 +1,124 @@
+                    all_values = all_values + str(k) + f" tuple<{val[val.find('[') + 1:val.rfind(']')]}>,"
+                elif issubclass(v.__origin__, FrozenSet):
+                    all_values = all_values + str(k) + f" frozen <set<{val[val.find('[') + 1:val.rfind(']')]}>>,"
+        return all_values[:-1]
+
+    @staticmethod
+    def create_table(name: str, definition: dict) -> None:
+        # StorageObj for now
+        ksp, table = extract_ksp_table(name)
+        query_keyspace = "CREATE KEYSPACE IF NOT EXISTS %s WITH replication = %s" % (ksp, config.replication)
+        config.session.execute(query_keyspace)
+
+        primary_keys = definition['value_id']
+        columns = definition['fields']
+
+        pks = str.join(',', primary_keys.keys())
+        if definition["type"] is numpy.ndarray:
+            pks = "(storage_id, cluster_id),block_id"
+
+        all_keys = CqlCOMM.parse_definition_to_cass_format(primary_keys)
+        if columns:
+            all_cols = CqlCOMM.parse_definition_to_cass_format(columns)
+            total_cols = all_keys + ',' + all_cols
+        else:
+            total_cols = all_keys
+
+        query_table = "CREATE TABLE IF NOT EXISTS %s.%s (%s, PRIMARY KEY (%s));" \
+                      % (ksp,
+                         table,
+                         total_cols,
+                         pks)
+        try:
+            log.debug('MAKE PERSISTENCE: %s', query_table)
+            config.session.execute(query_table)
+        except Exception as ex:
+            log.warn("Error creating the StorageDict table: %s %s", query_table, ex)
+            raise ex
+
+    @staticmethod
+    def hcache_parameters_generator(ksp: str, table: str, object_id: UUID, keys: list, columns: list) -> tuple:
+        hcache_params = (ksp, table, object_id, [(-2 ** 63, 2 ** 63 - 1)], keys, columns,
+                         {'cache_size': config.max_cache_size,
+                          'writer_par': config.write_callbacks_number,
+                          'writer_buffer': config.write_buffer_size,
+                          'timestamped_writes': config.timestamped_writes})
+        return hcache_params
+
+    @staticmethod
+    def create_hcache(object_id: UUID, name: str, definition: dict) -> Union[object, HNumpyStore, Hcache]:
+        ksp, table = extract_ksp_table(name)
+        hcache_params = CqlCOMM.hcache_parameters_generator(ksp, table, object_id, list(definition["value_id"].keys()),
+                                                            list(definition["fields"].keys()))
+        if definition["type"] is numpy.ndarray:
+            return HNumpyStore(*hcache_params)
+        else:
+            return Hcache(*hcache_params)
+
+    @staticmethod
+    def delete_data(object_id):
+        res = config.execute(istorage_read_entry, [object_id])
+        if res:
+            res = res.one()
+        else:
+            raise ValueError("There are no records with the specified object_id")
```

Comment:
Parallel deletes should fail?
e.g. deleting the same object from different tasks